requirements/common.in

```diff
@@ -194,3 +194,6 @@ soupsieve
 
 # Circuit-breaking for outgoing services
 circuitbreaker
+
+# Runtime monkeypatching of django-stubs generics
+https://github.com/typeddjango/django-stubs/archive/c0414c7b025b77caf9f72dbaf9e8471e11aad085.zip#egg=django-stubs-ext&subdirectory=django_stubs_ext
```

Comment:
The egg specification needs a fake version number `#egg=django-stubs-ext==0.5.0+git` for [silly Pip-related reasons](https://github.com/pypa/pip/pull/10564#discussion_r857174063).